assets/src/dashboard/app/views/templates/components.js

```diff
@@ -127,14 +127,22 @@ export const NavButton = styled(Button)`
     display: block;
     align-self: center;
     min-width: 0;
-    height: 40%;
-    color: ${theme.colors.gray900};
+    height: 40px;
+    width: 40px;
+    border-radius: 50%;
+    color: ${theme.colors.gray600};
     background-color: transparent;
-    border: none;
+    border: ${theme.borders.transparent};
 
     &:hover, &:active, &:focus {
-      color: ${theme.colors.bluePrimary};
+      background-color: ${theme.colors.gray600};
+      color: ${theme.colors.white};
+      @media ${theme.breakpoint.largeDisplayPhone} {
+        color: ${theme.colors.gray900};
+        background-color: transparent;
+       }
     }
+    transition: background-color 300ms ease-in-out, color 300ms ease-in-out;
```

Comment:
Super duper nit, I generally organize styles so the direct styles are all clustered together, followed by the `&:` styles, and then child element styles.  So, can the `transition` style be moved above the `&:hover, ...` chunk?

Reply:
Totally! So, I wanted the transition to take effect if going from inactive to hover as well as from hover to inactive so i placed it outside of the `:&` grouping.

Reply:
Does the placement of `transition` affect what it transitions? 🤔 Or do you mean it communicates better it's affecting all the `background-color` changes by putting it at the bottom?

Reply:
maybe i'm misunderstanding :) 

So I put the transition outside of the pseudo classes so that it catches on both _in_ and _out_ of the state. The below is exagerated but is what happens if we do 
```
&:hover { 
transition 
}
``` 
instead of 
```
&:hover { 
}
transition 
```

![transition only on hover](https://user-images.githubusercontent.com/10720454/81975995-bcbbf600-95dc-11ea-9779-15c2077ff290.gif)

Reply:
Oh I mean:
```
export const NavButton = styled(Button)`
  ${({ theme }) => `
    ...
    transition: background-color 300ms ease-in-out, color 300ms ease-in-out;

    &:hover, &:active, &:focus {
      ...
    }
```
So above the &:hover code, but not within it.

Reply:
ohhhh! that makes so much more sense! sure - i'll update that :) that was a journey!